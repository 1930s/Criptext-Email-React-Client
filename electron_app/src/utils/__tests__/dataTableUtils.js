/* eslint-env node, jest */
const { formContactsRow } = require('./../dataTableUtils.js');

describe('data table utils:', () => {
  it('form contacts row', () => {
    const contacts = [
      'Gianni <gianni@criptext.com>',
      'erika@criptext.com',
      'pedro@criptext.com'
    ];
    const state = formContactsRow(contacts);
    expect(state).toMatchSnapshot();
  });
});
