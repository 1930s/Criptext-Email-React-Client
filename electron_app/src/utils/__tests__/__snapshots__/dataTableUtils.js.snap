// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`data table utils: form contacts row 1`] = `
Array [
  Object {
    "email": "gianni@criptext.com",
    "name": "Gianni",
  },
  Object {
    "email": "erika@criptext.com",
    "name": null,
  },
  Object {
    "email": "pedro@criptext.com",
    "name": null,
  },
]
`;
